CrosswordMe.py

```diff
@@ -45,29 +46,29 @@ def write_horizontal(words, word_index, puzzle):
             word_position += 1 #Fake loop for the word position
     return puzzle
 
-def write_vertical(words, word, puzzle):
-    if word == 0:
+def write_vertical(words, word_index, puzzle):
+    if word_index == 0:
 
-        start_x_position, start_y_position, end_x_position, end_y_position = calculate_postions(len(words[word]))
+        start_x_position, start_y_position, end_x_position, end_y_position = calculate_postions(len(words[word_index]))
         column = randint(0, column_count-1)
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/CrosswordMe.py
+++ b/CrosswordMe.py
@@ -50,15 +50,16 @@
     if word_index == 0:
 
         start_x_position, start_y_position, end_x_position, end_y_position = calculate_postions(len(words[word_index]))
-        column = randint(0, column_count-1)
+        column = randint(0, column_count - 1)
 
-        word_position = 0 #resets the word_position. this is needed beacuse word_position counts through the length of every word being written
-        for array_position in range(start_y_position, end_y_position): #Loops through position within the array
+        word_position = 0  # resets the word_position. this is needed beacuse word_position counts through the length of every word being written
+        for array_position in range(start_y_position, end_y_position):  # Loops through position within the array
 
-            puzzle[array_position][column] = words[word_index][word_position] #Writes the word at random position of the array
-            word_position += 1 #Fake loop for the word position
+            puzzle[array_position][column] = words[word_index][word_position]  # Writes the word at random position of the array
+            word_position += 1  # Fake loop for the word position
 
     return puzzle
+
 
 def write_diagonal(words, word_index, puzzle):
     if word_index == 0:
```